### PR TITLE
Added recipe for bson package

### PR DESCRIPTION
**Pull request by ChrisBarker-NOAA**
Straighforward PyPi recipe

### CODE REVIEW

File: recipes/bson/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 1
```
Comment (synapticarbors) [inline recipes/bson/meta.yaml]: build number `0`.

File: recipes/bson/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+about:
+  summary: Independent BSON codec for Python that doesn't depend on MongoDB.
+  home: https://github.com/py-bson/bson
+  license: BSD-3-Clause
+  license_family: BSD
+
+extra:
+  recipe-maintainers:
+    - robmcmullen
```
Comment (synapticarbors) [inline recipes/bson/meta.yaml]: @robmcmullen -- are you ok being added as a co-maintainer of the `bson` package on conda-forge?

Comment (robmcmullen) [inline recipes/bson/meta.yaml]: OK with me.

File: recipes/bson/meta.yaml (hunk 1 above)
Comment (ChrisBarker-NOAA) [inline recipes/bson/meta.yaml]: Well, it's the zeroth build on conda-forge, but not the zeroth on Anaconda.org.

But sure, it's not different in any meaningful way.

**Pull request merged**